fast64_internal/oot/room/properties.py

```diff
@@ -49,6 +49,9 @@ def draw_props(self, layout: UILayout, headerIndex: int, index: int, objName: st
 
         if isLegacy:
             objectName = ootData.objectData.ootEnumObjectIDLegacy[self["objectID"]][1]
+        elif self.objectKey == "":
+            # just in case there's invalid objects keys, allows users to delete them
+            objectName = "DELETE ME!"
```

Comment:
since `""` is not a valid value for objectKey I don't understand how this can happen

The id for deleted entries (see `self.deletedEntry` in `OoT_ObjectData.__init__`) is `"None"`, maybe that's what should be checked here instead?
Or `self.objectKey not in ootData.objectData.objectsByKey` ?